src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -581,7 +581,8 @@ extern "C" int32_t GlobalizationNative_GetLatestJapaneseEra()
     if (U_FAILURE(err))
         return 0;
 
-    int32_t ret = ucal_getLimit(pCal, UCAL_ERA, UCAL_MAXIMUM, &err);
+    ucal_set(pCal, UCAL_EXTENDED_YEAR, 9999);
```

Comment:
Nit, this could maybe use a comment, as most likely future developers will have to dig up the change description to understand why 9999 is used here.

No need to reset the PR though.

Reply:
Or is it possible to add a test for this? That's even better than a comment, if possible.

Reply:
We already have many test for this area. The only thing I cannot add test for is using future era (from ICU). but I have tested this manually. Our test coverage is fairly good in this area.